s3sql/cli.py

```diff
@@ -44,12 +46,20 @@ def detect_file(ext):
         return {'ext':'.parquet','read_method':'read_parquet'}
     else:
         return "Read file type not supported, please try again with either a .csv, .json, or .parquet file extension."
+    
+def get_version():
+    return version("s3sql")
```

Comment:
FYI I was getting the following error, when attempting to run:

```
UnboundLocalError: cannot access local variable 'version' where it is not associated with a value
```

So I believe we need to assign to a variable first, and then return the value:

```suggestion
    v = version("s3sql")
    return v
```